app/templates/main/request-received.html

```diff
@@ -0,0 +1,43 @@
+{% extends "base.html" %}
+{% block beforeContent %}
+{{ super() }}
+{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <div class="govuk-panel govuk-panel--confirmation">
+            <h1 class="govuk-panel__title">
+                Request received
+            </h1>
+        </div>
+        <h2 class="govuk-heading-m govuk-!-margin-top-7">
+            What happens next
+        </h2>
+        <p class="govuk-body">
+            We will email a link to
+            <span class="govuk-!-font-weight-bold">
+            {{user_email}}
+            </span>.
```

Comment:
```suggestion
            We will email a link to
            <span class="govuk-!-font-weight-bold">{{user_email}}</span>.
```

This needs to be collapsed onto a single line to make sure that there's no space between the email address and the full stop please :)

### Before
<img width="450" alt="image" src="https://github.com/communitiesuk/funding-service-design-post-award-data-frontend/assets/2920760/470c3b63-fd26-46f2-9570-8d37c6f456db">


### After
<img width="454" alt="image" src="https://github.com/communitiesuk/funding-service-design-post-award-data-frontend/assets/2920760/cf59b8b5-8557-4f88-ae27-b257378e6f63">